tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -167,6 +169,22 @@
 # It is quite clear that the MRI surfaces (head, brain) are not well aligned
 # to the head digitization points (dots).
 #
+# Compute the transform
+# ---------------------
+# Let's step through the process that the transform is accomplishing.
+
+head_space = np.array([dig['r'] for dig in
+                       raw.info['dig']], dtype=float)
+
+# Transform digitization points into MRI space, adjust units by * 1e3 mm -> m
```

Comment:
to me it's the opposite. if you * 1e3 you go from m to mm

@drammock should this be added here?

Reply:
Yes, my mistake. Also I am not sure if this is useful for people who would be interested in this example. Maybe if I implement @jasmainak's suggestion it will be better.